packages/admin/admin/src/common/Accordion.tsx

```diff
@@ -0,0 +1,98 @@
+import ArrowForwardIosSharpIcon from "@mui/icons-material/ArrowForwardIosSharp";
+import MuiAccordion from "@mui/material/Accordion";
+import MuiAccordionDetails, { AccordionDetailsProps } from "@mui/material/AccordionDetails";
+import MuiAccordionSummary, { AccordionSummaryProps } from "@mui/material/AccordionSummary";
+import { styled } from "@mui/material/styles";
+import * as React from "react";
+
+interface AccordionComponentsProps {
+    accordionSummary?: Partial<AccordionSummaryProps>;
+    accordionDetails?: Partial<AccordionDetailsProps>;
+}
+interface AccordionProps {
+    title: React.ReactNode;
+    supportText?: React.ReactNode;
+    endAdornment?: React.ReactNode;
+    initialExpanded?: boolean;
+    componentsProps?: AccordionComponentsProps;
+}
+
+export const Accordion: React.FC<AccordionProps> = ({ title, supportText, endAdornment, children, initialExpanded = false, componentsProps }) => {
+    const [expanded, setExpanded] = React.useState(initialExpanded);
+
+    const handleChange = (event: React.SyntheticEvent, isExpanded: boolean) => {
+        setExpanded(isExpanded);
+    };
+
+    return (
+        <AccordionComponent expanded={expanded} onChange={handleChange} disableGutters elevation={0} square>
```

Comment:
Should the props `disableGutters`, `elevation` and `square` maybe be set in the default-props of the `MuiAccordion` component? 

Similar to what we have done for `MuiPaper`: https://github.com/vivid-planet/comet/blob/main/packages/admin/admin-theme/src/componentsTheme/MuiPaper.ts#L6